lib/tasks/coupons.rake

```diff
@@ -0,0 +1,24 @@
+namespace :coupons do
```

Comment:
it's not necessary to have a single file for a namespace. these rake tasks could have been in different files as we.. same file is also fine for this case

Reply:
Ok Sir